pkg/vdr/fingerprint/fingerprint.go

```diff
@@ -19,13 +19,22 @@ import (
 
 const (
 	// source: https://github.com/multiformats/multicodec/blob/master/table.csv.
-	ed25519pub = 0xed // Ed25519 public key in multicodec table
+	ed25519pub    = 0xed   // Ed25519 public key in multicodec table
+	bls12381g2pub = 0xeb   // BLS12-381 G2 public key in multicodec table
+	p256pub       = 0x1200 // NIST P-256 public key in multicodec table
+	p384pub       = 0x1201 // NIST P-384 public key in multicodec table
+	p521pub       = 0x1202 // NIST P-521 public key in multicodec table
```

Comment:
Constants should be exported. e.g `CreateDIDKeyByCode(P256pub,key)`

Reply:
contents are now exported